packages/website/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@lowdefy/website",
+  "version": "4.0.0-alpha.37",
+  "license": "Apache-2.0",
+  "description": "",
+  "homepage": "https://lowdefy.com",
+  "keywords": [
+    "lowdefy",
+    "documentation",
+    "website"
+  ],
+  "bugs": {
+    "url": "https://github.com/lowdefy/lowdefy/issues"
+  },
+  "contributors": [
+    {
+      "name": "Sam Tolmay",
+      "url": "https://github.com/SamTolmay"
+    },
+    {
+      "name": "Gerrie van Wyk",
+      "url": "https://github.com/Gervwyk"
+    },
+    {
+      "name": "Stephanie Smit",
+      "url": "https://github.com/StephanieJKS"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lowdefy/lowdefy.git"
+  },
+  "type": "module",
+  "exports": {
+    "./actions": "./dist/actions.js",
+    "./operators/client": "./dist/operatorsClient.js",
+    "./types": "./dist/types.js"
+  },
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "build": "swc src --out-dir dist --config-file ../../.swcrc --delete-dir-on-start",
+    "clean": "rm -rf .lowdefy",
+    "prepublishOnly": "pnpm build",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
```

Comment:
Please remove these scripts as well as `dependencies` and `devDependencies` and `exports`, because they are not used. Unless we are planning to use custom plugins for the website, in which case they should stay

Removing the test script will make the github workflows tests pass, since Jest fails if it doesn't find any tests

Reply:
We actually maybe don't need a `package.json` at all if we are not using plugins, this might be better because then then we won't publish an empty package to npm

Reply:
We might need a custom plugin or two for a few of the components that still need to be added. 

Please see 5cfff352b93631dd86ec75f7dab0aed09370df31